package.json

```diff
@@ -43,15 +42,14 @@
     "@cspotcode/source-map-support": "^0.8.1"
   },
   "devDependencies": {
+    "@danielx/civet": "^0.6.26",
```

Comment:
I'd feel safer if this were a pinned dep (no `^`). Given how we tend to change things and just bump the patch number. 😅

Reply:
I believe the lockfile should take care of it but being explicit here would also be fine.